README.rst

```diff
@@ -123,11 +126,36 @@ The example produces a file greeting all names included in the
      Hello Jane Doe!
      Hello Joe Bloggs!
 
-Running the example on REANA cloud
+Running the above example on REANA
 ==================================
 
-We are now ready to run this example and on the `REANA <http://www.reana.io/>`_
-cloud.
+We are now ready to run this example and on the `REANA <http://www.reana.io/>`_ platform.
+
+There are two ways to execute this analysis example on REANA.
+
+If you would like to simply launch this analysis example on the REANA instance
+at CERN and inspect its results using the web interface, please click on one of
+the following badge:
+
+.. image:: https://img.shields.io/badge/Launch%20with%20Serial%20on-REANA@CERN%20-ff3336
+   :target: https://reana.cern.ch/launch?url=https%3A%2F%2Fgithub.com%2Freanahub%2Freana-demo-helloworld&specification=reana.yaml
+
+.. image:: https://img.shields.io/badge/Launch%20with%20Yadage%20on-REANA@CERN%20-ff3336
```

Comment:
Four badges in the same row may be perhaps too much.

What about 2x2 setup?

Or each badge in a new line?

Have you tried how does it look?

Reply:
No limit:

![Screenshot 2022-04-07 at 17 51 09](https://user-images.githubusercontent.com/15694232/162244415-d5d44d5d-2055-4cd2-944e-57bc24fe572c.png)

Reply:
It looks like a full new paragraph, so the spacing is too big:

![b](https://user-images.githubusercontent.com/517546/162471145-b6641957-52b5-4580-99c4-50195cf71643.png)

Perhaps try `raw` HTML so that the lines would be closer?

(Otherwise I like images being on new lines; it gives better overview of what's possible when compared to free no-limit display that looks more messy, as it were.)

Reply:
Also, please order the badges alphabetically, to be preference neutral -- i.e. CWL, Serial, Snakemake, Yadage -- as also listed in the introductory phrase.